src/tests/JIT/SIMD/Directory.Build.props

```diff
@@ -5,17 +5,19 @@
     <CLRTestBatchPreCommands>
       <![CDATA[
 $(CLRTestBatchPreCommands)
-set DOTNET_JitFuncInfoLogFile=SIMD.log
 if EXIST SIMD.log (del SIMD.log)
 ]]>
     </CLRTestBatchPreCommands>
-    <BashCLRTestPreCommands>
+    <CLRTestBashPreCommands>
       <![CDATA[
-$(BashCLRTestPreCommands)
-export DOTNET_JitFuncInfoLogFile=SIMD.log
+$(CLRTestBashPreCommands)
 rm -f SIMD.log
 ]]>
-    </BashCLRTestPreCommands>
+    </CLRTestBashPreCommands>
   </PropertyGroup>
 
+  <ItemGroup>
+    <CLRTestBashEnvironmentVariable Include="COMPlus_JitFuncInfoLogFile" Value="SIMD.log" />
```

Comment:
Why did you revert this from DOTNET_ to COMPlus_?

Reply:
Thanks for catching this.  I think what happened is (1) I started this change before the COMPlus_ rename, (2) it has been open a long time due to it not being my highest priority and test failures (some real, some not), (3) I missed it when I merged the rename, and (4) my re-checks for envvar, complus_, precommands were limited to *proj and missed this file.  I checked again and also found some envvars in the new InstrumentedTiers.csproj (which I have less excuse for since I had already renamed the PreCommands variable in it...)